lib/services/user-service.ts

```diff
@@ -68,6 +73,15 @@ export class UserService implements IUserService {
 		this.$fs.deleteFile(this.getUserFilePath());
 	}
 
+	public async getUserAvatar(): Promise<string> {
+		if (!this.hasUser()) {
+			return null;
+		}
+
+		const userInfo = await this.$authCloudService.getUserInfo();
+		return userInfo.externalAvatarUrl;
```

Comment:
It seems like the gravatar is not used and there is no UI to set it. Also, the avatarType is some magic int with the only known value of `2` which mean `External`